azure_functions_worker/utils/wrappers.py

```diff
@@ -45,6 +45,7 @@ def call(*args, **kwargs):
             except expt_type as e:
                 if debug_logs is not None:
                     logger.error(debug_logs)
+                error_logger.exception("Error: %s, %s", e, message)
```

Comment:
exception vs error - how is it handled within Python and Host and how does it show up in Appinsights?

Reply:
Both are treated as errors but logger.exception gives the entire stack trace while error just gives the logger.error message.